docs/build.gradle

```diff
@@ -19,6 +19,169 @@
 
 apply plugin: 'elasticsearch.docs-test'
 
+/* Lower this number as you convert snippets to `// CONSOLE` and
+ *`// TESTRESPONSE`. Try not to raise it.... */
+buildRestTests.expectedUnconvertedCandidates = [
+  'plugins/discovery-azure-classic.asciidoc',
```

Comment:
I considered reading these from a file instead but decided it wasn't worth the effort because we should convert all the files before too long, removing the need for the list.